packages/main/src/plugin/color-registry.ts

```
@@ -549,23 +555,28 @@ export class ColorRegistry {
     this.registerColor(`card-bg`, {
       dark: charcoal[800],
       light: gray[300],
-      hcDark: black,
+      hcDark: gray[900],
       hcLight: gray[100],
     });
 
     this.registerColor(`card-header-text`, {
```

Biome (2.4.9)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)